src/AMAMM.sol

```diff
@@ -189,13 +204,10 @@ contract AMAMM is IAmAmm {
 
     /// @dev Returns whether the am-AMM is enabled for a given pool
     function _amAmmEnabled(PoolId id) internal view virtual returns (bool) {
-        //TODO
-        return true;
+        return enabled[id];
     }
 
-    function claimFees(Currency currency, address recipient) external override returns (uint256 fees) {}
-
-    function _pullBidToken(PoolId id, address from, uint256 amount) internal virtual {}
-
-    function _pushBidToken(PoolId id, address to, uint256 amount) internal virtual {}
+    function setEnabled(PoolId id, bool value) external {
+        enabled[id] = value;
```

Comment:
``` suggestion
        enabled[id] = !enabled[id];
```
Enhancement: The common practice is using a toggle. Also, it would require a permission check so that not everyone could call the function.